config/initializers/acuant_simulator_config_validation.rb

```diff
@@ -0,0 +1,37 @@
+##
+# This initializer is present to force an upgrade from the `acuant_simulator`
+# config to the `doc_auth_vendor` config. It can be removed after a deploy
+# cycle where everyone has had a chance to upgrade to the new config.
+#
+# Flipping the FORCE_ACUANT_CONFIG_UPGRADE below will require prevent the app
+# from starting if the acuant simulator is enabled but the mock doc auth vendor
+# vendor is not turned on.
+#
+FORCE_ACUANT_CONFIG_UPGRADE = false
```

Comment:
I like this aggressive enforcement of configs!

Reply:
Trying to get ahead of questions about why it suddenly quit working.